atlp-rwanda/Dominators-Phantom-FE
# Feature in app notification to drivers bus

# In app notification PR
## As an **operator** 
I **should be able to** send a driver a realtime notification when I assign/un assign them to/from a bus
**So that** they are immediately aware of the changes

## Extra notes
- Please add an implementation plan in the activity section hereunder, could be an algorithm/pseudo code. Example for login feature:
  1. check validity of email and password
  2. if i, check if email exists
  3. if ii, compare for equality, received and saved password hashes
  4. if iii, generate a token and return in response
  5. if not i or ii or iii, return response with appropriate error message 
- You will also attach any PRs and succeeding conversations in the same section
## Task descriptions

1. An in-app notification is sent to the target driver upon successful assignment to a bus
2.  An in-app notification is sent to the target driver upon successful un assignment from a bus
3. All notifications are saved to database so absentee targets can retrieve their notifications when back online
### How to test this PR
1. first of all check in `.nev.example` copy those values from their paste it in `.env`
2. after use this command `npm install`
3. then start by assign driver to bus after if you did well assigned driver will receive notification in time
